### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
@jimbair PTAL :wink:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# rpmdeplint container image
+
+This repository contains bits needed to build a container image for the [rpmdeplint](https://pagure.io/rpmdeplint). This image can be later used by the Fedora CI.
+
+## Development
+
+For testing purposes, you can either build the image locally or you can open a pull request in this repository and CI will build the image for you.
```
Comment (sturivny) [inline README.md]: repository, and CI

File: README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  --task-id TASK_ID, -t TASK_ID
+                        Koji task id
+  --release RELEASE, -r RELEASE
+                        release, like f31
+
+```
+
+### Note about promoting to production
+
+Merging pull requests to master branch also triggers build in CI. However, such images are **not*** automatically promoted and used by the CI pipelines. In order to promote a new image to production, you need to explicitly say so in the [rpmdeplint-pipeline](https://github.com/fedora-ci/rpmdeplint-pipeline/README.md#promoting-new-rpmdeplint-image-to-production) repository.
```
Comment (sturivny) [inline README.md]: to the master

**Review by sturivny — COMMENTED** [review]
LGTM

**Comment by msrb** [comment]
Thanks @sturivny :wink:

File: README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# rpmdeplint container image
+
+This repository contains bits needed to build a container image for the [rpmdeplint](https://pagure.io/rpmdeplint). This image can be later used by the Fedora CI.
```
Comment (jimbair) [inline README.md]: I would say "to build a container image for the [rpmdeplint] test" or simply "to build a container image for [rpmdeplint]"

Also, I would say "This image can be later used by the Fedora CI project/team"  or "This image can be later used by Fedora CI."

File: README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Development
+
+For testing purposes, you can either build the image locally or you can open a pull request in this repository, and CI will build the image for you.
+
+To build the image locally, type:
```
Comment (jimbair) [inline README.md]: Replace "type" with "run" I think

File: README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  --task-id TASK_ID, -t TASK_ID
+                        Koji task id
+  --release RELEASE, -r RELEASE
+                        release, like f31
+
+```
+
+### Note about promoting to production
+
+Merging pull requests to the master branch also triggers build in CI. However, such images are **not*** automatically promoted and used by the CI pipelines. In order to promote a new image to production, you need to explicitly say so in the [rpmdeplint-pipeline](https://github.com/fedora-ci/rpmdeplint-pipeline/README.md#promoting-new-rpmdeplint-image-to-production) repository.
```
Comment (jimbair) [inline README.md]: Small change:
"Merging pull requests to the master branch also triggers a build in CI."

File: README.md (hunk 3 above)
Comment (msrb) [reply README.md]: Updated, thanks ;)

File: README.md (hunk 5 above)
Comment (msrb) [reply README.md]: Updated, thanks ;)

File: README.md (hunk 4 above)
Comment (msrb) [reply README.md]: Updated, thanks ;)

**Comment by msrb** [comment]
Thanks for the review @sturivny and @jimbair :)

**Pull request merged**